### PR TITLE
CCG remove traditional clients feature #18879 (the rest)

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/18879

*Before you submit this PR make sure you have added an entry to **.changelog** file.*

# Description

Short summary of why you created this PR (if you added documentation, please add any relevant diagram).

# Target branches

* Which documentation version does this PR apply to?
* Does this content need to be backported? If this is the case, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

- [ ] master
- [ ] manager-4.3

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/registration-cli.adoc [hunk 1]
```diff
@@ -71,7 +71,7 @@ salt-key -a '<client>'
 --
 
 
-=== Salt Minion configuration
+== Salt Minion configuration
```
Comment (0rnela) [inline modules/client-configuration/pages/registration-cli.adoc:74]:
```suggestion
== {salt} Minion configuration
```

**Review by 0rnela — CHANGES_REQUESTED** [review]

File: modules/client-configuration/pages/repositories.adoc [hunk 2]
```diff
@@ -25,7 +25,7 @@ This can lead to installing unexpected packages.
 
 Local repositories are disabled during onboarding.
 
-For Salt clients, local repositories will be disabled each time a channel state is executed.
+Local repositories will be disabled each time a channel state is executed.
 For example, when applying the highstate or performing a package action.
```
[the remark this answers is not in the document]
Comment (keichwa) [reply modules/client-configuration/pages/repositories.adoc:29]: I think we better should keep sentences short.

File: modules/client-configuration/pages/registration-cli.adoc [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [[registering-clients-cli]]
-= Register on the Command Line (Salt)
+= Register on the Command Line
 
 == Manual Salt client registration
```
Comment (0rnela) [inline modules/client-configuration/pages/registration-cli.adoc:4]:
```suggestion
== Manual client registration
```

File: modules/client-configuration/pages/registration-cli.adoc [hunk 4]
```diff
@@ -71,9 +71,9 @@ salt-key -a '<client>'
 --
 
 
-=== Salt Minion configuration
+== {salt} Minion configuration
```
Comment (0rnela) [inline modules/client-configuration/pages/registration-cli.adoc:74]:
```suggestion
== Client configuration
```

File: modules/client-configuration/pages/registration-cli.adoc [hunk 5]
```diff
@@ -71,9 +71,9 @@ salt-key -a '<client>'
...
 
-Salt Minion ([package]``salt-minion``)::
+{salt} Minion ([package]``salt-minion``)::
```
Comment (0rnela) [inline modules/client-configuration/pages/registration-cli.adoc:76]:
```suggestion
Client ([package]``salt-minion``)::
```

File: modules/client-configuration/pages/system-types.adoc [hunk 6]
```diff
@@ -4,77 +4,10 @@
 Clients are categorized by system type.
 Every client can have both a base system type, and an add-on system type assigned.
 
-Base system types include ``Management``, for traditional clients, and ``Salt`` for Salt clients.
+Base system types include ``Salt`` for Salt clients.
```
Comment (0rnela) [inline modules/client-configuration/pages/system-types.adoc:7]:
```suggestion
Base system type is ``Salt`` for all clients.
```

**Review by 0rnela — CHANGES_REQUESTED** [review]

**Pull request merged**